pom.xml

```diff
@@ -22,38 +22,47 @@
     <modelVersion>4.0.0</modelVersion>
     <parent>
         <groupId>org.apache.sling</groupId>
-        <artifactId>sling</artifactId>
-        <version>16</version>
-        <relativePath/>
+        <artifactId>sling-bundle-parent</artifactId>
+        <version>47</version>
+        <relativePath />
     </parent>
 
     <artifactId>org.apache.sling.commons.cache.api</artifactId>
     <version>0.1-SNAPSHOT</version>
-    <packaging>bundle</packaging>
 
     <name>Apache Sling Commons Cache API</name>
     <description>
-        This bundle porvides a Cache API for Sling applications to use.
+        This bundle provides a Cache API for Sling applications to use.
     </description>
 
+    <properties>
+        <sling.java.version>8</sling.java.version>
+        <project.build.outputTimestamp>1</project.build.outputTimestamp>
+        <bnd.baseline.skip>true</bnd.baseline.skip>
+    </properties>
+
     <scm>
         <connection>scm:git:https://gitbox.apache.org/repos/asf/sling-org-apache-sling-commons-cache-api.git</connection>
         <developerConnection>scm:git:https://gitbox.apache.org/repos/asf/sling-org-apache-sling-commons-cache-api.git</developerConnection>
         <url>https://gitbox.apache.org/repos/asf?p=sling-org-apache-sling-commons-cache-api.git</url>
+        <tag>HEAD</tag>
     </scm>
-    
 
     <build>
         <plugins>
             <plugin>
-                <groupId>org.apache.felix</groupId>
-                <artifactId>maven-bundle-plugin</artifactId>
-                <version>2.3.6</version>
-                <extensions>true</extensions>
-                <configuration>
-                    <instructions>
-                    </instructions>
-                </configuration>
+                <groupId>biz.aQute.bnd</groupId>
+                <artifactId>bnd-maven-plugin</artifactId>
+                <executions>
+                    <execution>
+                        <id>bnd-process</id>
+                        <configuration>
+                            <bnd><![CDATA[
+                            Export-Package: org.apache.sling.commons.cache.api;version="1.0"
```

Comment:
Every versioned package is exported by default. Please use OSGI versioning annotations on package level instead

Reply:
Fixed